c_check

```diff
@@ -9,7 +9,7 @@ $hostarch = `uname -m | sed -e s/i.86/x86/`;chop($hostarch);
 $hostarch = `uname -p` if ($hostos eq "AIX");
 $hostarch = "x86_64" if ($hostarch eq "amd64");
 $hostarch = "arm" if ($hostarch =~ /^arm.*/);
-$hostarch = "arm64" if ($hostarch eq "aarch64");
+$hostarch = "arm64" if ($hostarch =~ /^(aarch64|arm64).*/);
```

Comment:
Shouldn't this go above Line 11?

Reply:
Depends on how perl handles reassignment, there is no exit after first match

Reply:
I'm bad at perl, but if `hostarch` is initially set to `arm64`, I suppose:
- line 11 will set it to `arm` (because `arm64` matches `/^arm.*/`)
- line 12 will not match, and do nothing

Reply:
Ugh- did not notice (on phone screen but that is a lame excuse) that it is reading and writing the same variable.better bury this PR quickly then